cmd/root.go

```diff
@@ -78,8 +60,85 @@ func initConfig() {
 
 	viper.AutomaticEnv()
 
+	viper.OnConfigChange(func(e fsnotify.Event) {
+		fmt.Printf("Config file changed: %s\n", e.Name)
+
+		var newConfig *BackendConfig
+		viper.Unmarshal(&newConfig)
+
+		// used because OnConfigChange runs twice for some reason
+		// allows to skip empty first config
+		isEmpty := isConfigEmpty(reflect.ValueOf(*newConfig))
+		if isEmpty {
+			return
+		}
+
+		SetBackendConfigDefaults(newConfig)
+
+		isEqual := reflect.DeepEqual(*backendConfig, *newConfig)
+		if !isEqual {
```

Comment:
Should we add an else that alerts the user that the config change wasn't reloadable with a log?